packages/react/src/components/Card/CardWrap.js

```diff
@@ -1,10 +1,9 @@
-import { createElement } from 'react'
 import styled, { css } from 'styled-components'
+import { createElement } from 'react'
 
 import { media, isLarge } from '../../utils'
 
 const HEIGHT = '382px'
-const CACHE = {}
```

Comment:
@breadadams I removed this because I think styled-components do that internally.